www/js/diary/list.js

```diff
@@ -100,14 +116,16 @@ angular.module('emission.main.diary.list',['ui-leaflet', 'nvd3ChartDirectives',
       todayButtonType : 'button-assertive',  //Optional
       closeButtonType : 'button-assertive',  //Optional
       inputDate: new Date(),  //Optional
+      from: new Date(2015, 1, 1),
+      to: new Date(),
       mondayFirst: true,  //Optional
       templateType: 'popup', //Optional
       showTodayButton: 'true', //Optional
       modalHeaderColor: 'bar-positive', //Optional
       modalFooterColor: 'bar-positive', //Optional
       callback: $scope.setCurrDay, //Mandatory
-      dateFormat: 'dd-MMMM-yyyy', //Optional
-      closeOnSelect: true, //Optional
+      dateFormat: 'dd MMMM yyyy', //Optional
```

Comment:
should we change this to be yyyy MMMM dd, similar to the button that launches the selector?